kubevirt-vmware/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	ovirtsdk "github.com/ovirt/go-ovirt"
+)
+
+func main() {
+	ca := `-----BEGIN CERTIFICATE-----
```

Comment:
this does not seem like a content we should be commiting :)

Reply:
my test file should not be there :)